packages/components/src/components/input-field/input.jsx

```diff
@@ -24,6 +24,7 @@ const Input = ({
     is_read_only,
     max_length,
     name,
+    onBlurHandler,
```

Comment:
Could you call it `handleBlur` or `onBlur` instead and help to move it to a proper place alphabetically.

Reply:
Will rename it